### PR TITLE
Feature: funding progress bar

### PR DESCRIPTION
**Pull request by edbrett**
D3 x axis chart to showing the current progress of funding using the funding progress data from carto:
- Fetch data from carto by target type
- Plot d3 x axis with current amount and milestones along axis
- style svg container for responsive, removing aspect ratio for texts

other: add global map to regions page, small style fixes

### CODE REVIEW

File: app/assets/javascripts/views/funding_progress.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        target: this.status.get('target')
+      };
+
+      this.model.getFundingProgress(params).done(function(){
+        this.render();
+      }.bind(this));
+    },
+
+    _addListeners: function() {
+      this.status.on('change:target', this._fetchData.bind(this));
```
Comment (geriux) [inline app/assets/javascripts/views/funding_progress.js]: Let's start using `listenTo` for new views =D

**Review by geriux — CHANGES_REQUESTED** [review]
Adding a [task](https://www.pivotaltracker.com/story/show/137008617) on Pivotal to check some duplicated events that are being triggered

**Comment by edbrett** [comment]
Listeners had been set in the parent view as well :|. Removed. Also update to listenTo on model.

**Review by geriux — APPROVED** [review]

**Comment by geriux** [comment]
Awesome! Thanks! Merging!

**Pull request merged**